src/server/sql/preferences/create_preferences_table.sql

```diff
@@ -9,5 +9,7 @@ CREATE TABLE IF NOT EXISTS preferences (
 	default_chart_to_render graph_type NOT NULL,
 	default_bar_stacking BOOLEAN NOT NULL,
 	default_language language_type NOT NULL,
-	default_timezone TEXT DEFAULT NULL 
+	default_timezone TEXT DEFAULT NULL,
+	default_warning_file_size VARCHAR(50) NOT NULL,
```

Comment:
I'm unclear on why these are a character type when it is a number.
Also, need a migration for the table change. I am pushing that out since OED does not have clear directions on doing this. Note it would need its type changed if it is changed here.

Reply:
I've changed the type to float in the new commit. Thank you for helping me create the migration for the table change.